api/Navigator.json

```diff
@@ -445,7 +445,7 @@
               "version_added": false
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": "79"
```

Comment:
Did you confirm it? Because we haven't added support for `navigator.canShare`.

Reply:
Hmmm. I ran http://wpt.live/web-share/canShare-files.tentative.https.html test on my phone with latest FF and it showed success with a single file. Given that we know files are not supported, this is perhaps a false positive.

I will revert that.

Reply:
Reverted.

Reply:
That test is a little confusing. It tests whether passing a file without wrapping it with an array fails or not, and it does fail because it's not supported at all anyway. 😄

Reply:
So much for being "clever" in my due diligence. In any case, this is now the most trivial review for you ever!